static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-liveblog-design-test.js

```diff
@@ -0,0 +1,151 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'lib/geolocation',
+    'lodash/utilities/template',
+    'lib/config',
+    'lib/$',
+    'raw-loader!common/views/acquisitions-epic-liveblog.html',
+    'raw-loader!common/views/acquisitions-epic-liveblog-old-design-subtle.html',
+    'raw-loader!common/views/acquisitions-epic-liveblog-old-design-minimal.html'
+], function (
+    contributionsUtilities,
+    geolocation,
+    template,
+    config,
+    $,
+    liveblogEpicTemplateControl,
+    liveblogEpicTemplateOldDesignSubtle,
+    liveblogEpicTemplateOldDesignMinimal
+) {
+
+    var pageId = config.page.pageId || '';
+
+    var insertEpicAfterSelector = '.js-insert-epic-after';
+
+    function isEpic(el) {
+        $(el).hasClass('is-liveblog-epic');
+    }
+
+    function getLiveblogEntryTimeData(el) {
+        var $timeEl = $('time', el);
+
+        return {
+            datetime: $timeEl.attr('datetime'),
+            title: $timeEl.attr('title'),
+            date: $timeEl.text(),
+            time: $('.block-time__absolute', el).text()
+        };
+    }
+
+    function getNextEpicElement(el) {
+        var $epic = $(el).next();
+
+        while ($epic.length && !isEpic($epic[0])) {
+            $epic = $epic.next();
+        }
+
+        if (!isEpic($epic[0])) {
+            return null;
+        }
+
+        return $epic[0];
+    }
+
+    function setEpicLiveblogEntryTimeData(el, timeData) {
+        var $epicTimeEl = $('time', el);
+        $epicTimeEl.attr('datetime', timeData.datetime);
+        $epicTimeEl.attr('title', timeData.title);
+        $epicTimeEl.text(timeData.date);
+        $('.block-time__absolute', el).text(timeData.time);
+    }
+
+    function copyLiveblogEntryTimeDataToEpic() {
+        $(insertEpicAfterSelector).each(function(el) {
+            var timeData = getLiveblogEntryTimeData(el);
+            var nextEpicElement = getNextEpicElement(el);
+            setEpicLiveblogEntryTimeData(nextEpicElement, timeData);
+        });
+    }
+
+    return contributionsUtilities.makeABTest({
+        id: 'AcquisitionsEpicLiveblogDesignTest',
+        campaignId: 'epic_liveblog_design_test',
+        campaignSuffix: pageId.replace(/-/g, '_').replace(/\//g, '__'),
+
+        start: '2017-04-01',
+        expiry: '2018-04-01',
+
+        author: 'Joseph Smith',
+        description: 'This places the epic underneath liveblog blocks which the author has specified in Composer should have an epic against them',
```

Comment:
This description is copied from the "production" liveblog test - it could be more informative